style.css

```diff
@@ -20,10 +29,49 @@ nav { line-height: normal; }
 
 :root {
     --header-height: 3em;
+    /* custom */
     --header-bg: #f8f8f8;
+    --body-bg: white;
+    --code-bg: #f3f3f3;
+    --text-color: black;
+    --text-alt-color: hsl(210, 100%, 30%);
```

Comment:
```suggestion
    --link-color: hsl(210, 100%, 30%);
```

Reply:
Changed, the variable name in the newest commit.